app/api/pet_chat.py

```
@@ -1,12 +1,65 @@
-    tags=["pet-chat"]
-)
\ No newline at end of file
+llm_api = APIRouter(tags=["pet-chat"])
+
+@llm_api.get("/pet/{pet_id}")
+async def get_pet_by_id(pet_id: int):
+    """MCP를 통해 펫 정보 조회"""
+    try:
+        pet_info = await get_pet_info(pet_id)
+        if not pet_info:
+            raise HTTPException(status_code=404, detail="Pet not found")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)